### DISCUSSION TITLE
Which wireless wall (scene) switch can bind to devices or groups?

### DISCUSSION THREAD

**Discussion by Anjerlaan** [Koenkk/zigbee2mqtt#11427] [Devices]
Hi,

Which wireless wall (scene) switch can bind to devices/groups?
my current TS0043 is not able to bind

**Comment by induna-crewneck** [comment]
Did you ever manage to bind the TS0043?

**Reply by Anjerlaan** [reply]
Nope, I bought an EcoDim

**Reply by Anjerlaan** [reply]
This one only can add groups and not devices
